.dir-locals.el

```diff
@@ -1,8 +0,0 @@
-((nil . ((eval . (progn
-                   (defun figwheel-repl ()
-                     (interactive)
-                     (run-clojure "lein figwheel ios"))
-                   
-                   (add-hook 'clojure-mode-hook #'inf-clojure-minor-mode))))))
-
-
```

Comment:
Just some emacs stuff I should not really have committed and I don't need it anymore in any case.